components/docker-auth/pkg/extension/acl_test.go

```diff
@@ -47,7 +52,7 @@ func createConn() bool {
 	dbConnection, err = sql.Open(dbDriver, dbUser+":"+dbPass+"@tcp("+host+":"+port+")/"+dbName)
 	if err != nil {
 		fmt.Println("Error while connecting to the database")
-		os.Exit(ErrorExitCode)
+		os.Exit(1)
```

Comment:
I think the previous way is more readable. When an os.Exist happens a person might not know exactly what 1 means

Reply:
Fixed by #159